src/db/gc/snapshot.rs

```diff
@@ -239,6 +277,44 @@ where
                         }
                     }
                 }
+
+                // Backfill new db records during snapshot export
+                if let Ok(db) = open_db(self.db_root_dir.clone(), &self.db_config) {
+                    if let Some(mem_db) = self.memory_db.write().take() {
+                        let count = mem_db.len();
+                        let approximate_heap_size = {
+                            let mut s = 0;
+                            for (_k, v) in mem_db.iter() {
+                                s += 64;
```

Comment:
What is `64`?

Reply:
That's the size of `Cid`
```rust
pub type Cid = CidGeneric<64>;
```

Reply:
Actually, it's better to use `std::mem::size_of::<Cid>()`